src/CoreBundle/Resources/public/themes/standard/css/simpleTree.css

```diff
@@ -111,34 +111,44 @@
 	width: 28px;
 	height: 11px;
 	cursor:pointer;
-	cursor:hand;
 }
-#treelegend {
-  width: 350px;
+
+.navigationTreeContainer {
+	display: table;
```

Comment:
This is tab indentation?

Reply:
The whole file seems to be tab-intended - shouldn't be changed in this PR (normally I'd avoid making changes to files belonging to external libs, but this file has custom changes anyway).